docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "SingleStoreDB",
+  "links": {
+    "docs": "https://docs.singlestore.com/",
+    "github": "https://github.com/singlestore-labs",
+    "website": "https://www.singlestore.com/",
+    "vendor_discussion": "",
+    "poc_github": "",
```

Comment:
Please provide a GitHub username for the point of contact who we can reach, regarding any changes or verification of items for the DB in this table